recipes/default.rb

```diff
@@ -1,9 +1,10 @@
 #
+# Author:: Ryan Hass <rhass@chef.io>
 # Author:: Sean OMeara <someara@opscode.com>
```

Comment:
Might wanna update sean's e-mail since he's not at Chef anymore. It's sean@sean.io